components/esp_lvgl_port/include/esp_lvgl_port_lv_blend.h

```diff
@@ -18,6 +18,19 @@ extern "C" {
 #warning "esp_lvgl_port_lv_blend.h included, but CONFIG_LV_DRAW_SW_ASM_CUSTOM not set. Assembly rendering not used"
 #else
 
+#include "lvgl.h"
+
+/**
+ * LVGL 9.2 changed the blending API compared to 9.1
+ * The types were moved to lv_draw_sw_blend_private.h
+ * and the leading underscore from type names was removed
+ */
+#if __has_include("draw/sw/blend/lv_draw_sw_blend_private.h")
+#include "draw/sw/blend/lv_draw_sw_blend_private.h"
+#define _lv_draw_sw_blend_fill_dsc_t lv_draw_sw_blend_fill_dsc_t
+#define _lv_draw_sw_blend_image_dsc_t lv_draw_sw_blend_image_dsc_t
+#endif
```

Comment:
Hi @kisvegabor,

We’re including this compatibility code so our blending acceleration can compile correctly for both LVGL 9.1 and 9.2. When this feature was first implemented, the blending interface was intended to be public, but it now appears to be treated as a private API. Could you please share your plans for the blending interface moving forward? Are there any intentions to expose it publicly, or should we plan for further adjustments if it remains private?

Thanks!

Reply:
Fair enough, thank you!